ice.go

```diff
@@ -141,3 +160,87 @@ func (c *ICECandidate) Unmarshal(raw string) error {
 
 	return nil
 }
+
+// GetICEParams extracts session-level ICE parameters
+func (s *SessionDescription) GetICEParams() ICESessionParameters {
+	ret := ICESessionParameters{}
+
+	var value string
+	var found bool
+
+	_, found = s.Attribute("ice-lite")
+	if found {
+		ret.Lite = true
+	} else {
+		ret.Lite = false
+	}
+	value, found = s.Attribute("ice-ufrag")
+	if found {
```

Comment:
Default value is "" so you don't need to explicitly set

Just to keep it terser/less for the reader to process
```
if value, found = s.Attribute("ice-ufrag"); found {
  ret.UserFragment = value
}
```